src/EmailfilingConst.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Civi\Emailfiling;
```

Comment:
Can you move this to a helper folder or service folder? 
The src folder format is more common with drupal modules. To follow civicrm format for extensions you would need to move this to the relevant folder and name the class using the folder path, this helps keep files more organised.